main.py

```diff
@@ -128,42 +113,42 @@ def main():
                         .output(output_path, vcodec="libx264", pix_fmt="yuv420p", acodec="aac", crf="23")
                         .run()
                     )
-        else:
-            print("ERROR: Path not valid.")
 
 
-def display_aspect_ratio(string):
-    # Function to get the DAR from a given aspect ratio string.
+def display_aspect_ratio(ratio_string):
+    # Get the numeric ratio from the aspect ratio string, e.g. "4:3" → 1.333(3)
 
-    vals = [int(val) for val in string.split(":")]
-    return vals[0] / vals[1]
+    numbers = ratio_string.split(':')
+    width = int(numbers[0])
+    height = int(numbers[1])
 
+    return width / height
 
-def aspect_ratio_and_resolution(file):
-    # Function to get the aspect ratio and resolution from a given path to a video.
 
-    file_metadata = FFProbe(file)
+def stream_aspect_ratio(file):
+    # Returns the aspect ratio expressed as a string, e.g. "4:3"
 
+    file_metadata = FFProbe(file)
     ratio = None
+
     for stream in file_metadata.streams:
         if stream.is_video():
             width, height = stream.frame_size()
 
             for resolution in RESOLUTIONS:
                 if RESOLUTIONS[resolution]["width"] == width:
                     ratio = RESOLUTIONS[resolution]["aspect_ratio"]
+                    return ratio
 
-            if not ratio:
-                print("Incompatible input dimensions.")
-                sys.exit()
-
-    return ratio, [width, height]
+    if not ratio:
+        print("Incompatible input dimensions.")
+        sys.exit()
 
 
 def watermark_path(ratio):
     # Function to get a path to a corresponding .png watermark given an aspect ratio string.
 
-    for name, data in RESOLUTIONS.items():
+    for name, _ in RESOLUTIONS.items():
```

Comment:
This is iterating over the `RESOLUTIONS` dict, `for key, value in dict` - but since we only care about the keys, I've just named the value `_` to indicate that it's unused.